ni_measurementlink_service/_internal/parameter/serialization_strategy.py

```diff
@@ -1,25 +1,24 @@
 """Serialization Strategy."""
 
-from typing import Any, Callable
+from typing import Any, Callable, Dict
 
 from google.protobuf import type_pb2
-from google.protobuf.internal import decoder
-from google.protobuf.internal import encoder
+from google.protobuf.internal import decoder, encoder
+from typing_extensions import TypeAlias
```

Comment:
Let's not add runtime dependencies on typing_extensions. Also, we don't need typing_extensions with newer Python versions.

```suggestion
if typing.TYPE_CHECKING:
    if sys.version_info >= (3, 10):
        from typing import TypeAlias
    else:
        from typing_extensions import TypeAlias
```

Reply:
I get these errors if I use this instead of importing TypeAlias. Should I just remove the 'if typing.TYPE_CHECKING`?
```bat
ni_measurementlink_service\_internal\parameter\serialization_strategy.py:18: in <module>
    WriteFunction: TypeAlias = Callable[[bytes], int]
E   NameError: name 'TypeAlias' is not defined
```

Reply:
https://docs.python.org/3/library/typing.html#typing.TYPE_CHECKING gives two solutions to this problem.